_data/other.yml

```diff
@@ -245,6 +245,14 @@ websites:
       twitter: indeed
       facebook: Indeed
 
+    - name: Instapaper
+      url: https://www.instapaper.com
+      twitter: instapaper
```

Comment:
I think that `twitter` should be changed from [`instapaper`](https://twitter.com/instapaper) to [`InstapaperHelp`](https://twitter.com/InstapaperHelp) because Instapaper says on their [help page](https://www.instapaper.com/help): _"You can also _[...]_ tweet at @InstapaperHelp with ideas, feedback, and questions not covered here."_ What do other people think?